.travis.yml

```diff
@@ -24,7 +24,7 @@ env:
     - INSTALL_TYPE="develop"
     - INSTALL_TYPE="sdist"
     - INSTALL_TYPE="wheel"
-    - INSTALL_DEPENDS="pip==18.1 setuptools==30.2.1"
+    - INSTALL_DEPENDS="pip==19.2.3 setuptools==41.2.0"
```

Comment:
The latest will be installed and tested in `CHECK_TYPE="test"`. This entry is here specifically to test `pip`'s ability to upgrade `setuptools` using `pyproject.toml` when a too-old version is present. So it's almost certainly a `pip` problem, not a `setuptools` problem.

Reply:
I believe you are right. I think this is a pipenv problem

Reply:
This is still pointed to the latest.